pom.xml

```diff
@@ -4,13 +4,22 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
```

Comment:
Unless you really need this I would suggest using the Spring Maven BOM as per: https://www.baeldung.com/spring-maven-bom (Like you've done with the Microsoft dependencies)